app/policies/status_policy.rb

```diff
@@ -12,9 +12,11 @@ def index?
   end
 
   def show?
-    return false if author.suspended?
-
-    if requires_mention?
+    if role.can?(:manage_reports) && record.reported?
+      true
+    elsif author.suspended?
+      false
+    elsif requires_mention?
```

Comment:
I'm afraid this could cause moderators to see posts from suspended authors in the non-admin interface, causing confusion. Maybe this should be a separate policy.